openedx/features/marketplace/tests/challenge_model_test.py

```diff
@@ -0,0 +1,62 @@
+from ddt import data, unpack, ddt
```

Comment:
Am I missing any overrride of djnago's define functionality? If not then we haven't customized anything from django's Model functionality, We should right any of these unit test.

Reply:
@ahmed-faraz46 just for the sake of coverage